### PR TITLE
feat: naive-ui

### PR DESCRIPTION
**Pull request by productdevbook**

### CODE REVIEW

File: modules/Naive UI.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Naive UI
+description: Nuxt 3 Naive UI library
```
Comment (Intevel) [inline modules/Naive UI.yml:2]: Do not advertize with a specific Nuxt version, see here:

https://nuxt.com/docs/guide/going-further/modules#do-not-advertize-with-a-specific-nuxt-version

File: modules/Naive UI.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+github: https://github.com/huntersofbook/naive-ui-nuxt
+website: https://github.com/huntersofbook/naive-ui-nuxt
+learn_more: ''
+category: Devtools
+type: 3rd-party
+maintainers:
+  - name: huntersofbook
+    github: huntersofbook
+compatibility:
+  nuxt: ^2.0.0
```
Comment (Intevel) [inline modules/Naive UI.yml:15]: The description says Nuxt 3, but compatibility is for Nuxt 2?

File: modules/Naive UI.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: Naive UI
+description: Nuxt 3 Naive UI library
+repo: huntersofbook/naive-ui-nuxt
+npm: '@huntersofbook/naive-ui-nuxt'
+icon: ''
```
Comment (Intevel) [inline modules/Naive UI.yml:5]: You've uploaded an Icon, but doesnt use it.

File: modules/naive-ui.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+
+name: Naive UI
+description: >-
+  A Vue 3 Component Library
+  Fairly Complete, Theme Customizable, Uses TypeScript, Fast
+  Kinda Interesting
```
Comment (Intevel) [inline modules/naive-ui.yml:6]: Im actually not sure if this fits in the description ^^

**Review by Intevel — CHANGES_REQUESTED** [review]
Randomly saw this PR and came across a few things.

I think you have accidentally uploaded the "Naive UI.yml" file as well, it seems also incorrect, and the file name can lead sometimes to issues with a space in it. 

The other file looks good. 💚

File: modules/naive-ui.yml (hunk 4 above)
Comment (productdevbook) [reply modules/naive-ui.yml:6]: Thank you i will check soon

**Pull request closed without merging**